lib/utils/index.js

```diff
@@ -860,6 +860,44 @@ module.exports = {
       })
   },
 
+  /**
+   * Get getter body from computed function
+   * @param {CallExpression} callExpression call of computed function
+   * @return {FunctionExpression | ArrowFunctionExpression | null} getter function
+   */
+  getComputedGetterBody(callExpression) {
```

Comment:
Can you rename it to include the meaning that it is a function that only accepts `computed (() => {})`?
Or change the argument to be acceptable and use this function in `getComputedProperties` as well.

```js
/**
  * @param {Expression} expr
  */
getComputedGetterBody(expr) {
}

// ...
// use
const getter = utils.getComputedGetterBody(node.arguments[0])
```